<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Panier</class>
 <widget class="QWidget" name="F_Panier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>614</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>503</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ImagesIcones.qrc">
    <normaloff>:/Paiement.png</normaloff>:/Paiement.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" rowspan="2">
    <widget class="QWidget" name="gridWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Historiques des paiements</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QRadioButton" name="rB_Familles">
           <property name="text">
            <string>Fami&amp;lles</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="rB_Les_deux">
           <property name="text">
            <string>Les deu&amp;x</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Mois :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="CBx_Mois"/>
         </item>
         <item row="11" column="0" colspan="2">
          <widget class="QPushButton" name="Bt_ImprimerFacture">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Imprimer facture</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Filtre des paiements :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QRadioButton" name="rB_Collectivites">
           <property name="text">
            <string>Collecti&amp;vités</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QTreeWidget" name="TW_Paiements">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Année :</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="CBx_Annee"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Panier</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="5" column="0">
       <widget class="QLabel" name="Lb_NumCheque">
        <property name="text">
         <string>Numéro du chèque :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Choisir le moyen de paiement :</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="Lb_Remarque">
        <property name="text">
         <string>Remarque :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="Lb_Banque">
        <property name="text">
         <string>Banque :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Total :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="Le_NomMembre">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QDateEdit" name="DtE_DateCheque"/>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QTableView" name="TbV_Panier">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="CBx_Banque"/>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="Le_Emetteur"/>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="SBx_Total">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <double>999999999.990000009536743</double>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="3">
       <widget class="QLineEdit" name="LE_Remarque"/>
      </item>
      <item row="9" column="3">
       <widget class="QPushButton" name="Bt_OK">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Valider</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="Lb_Emetteur">
        <property name="text">
         <string>Emetteur du chèque :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Membre concerné :</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="Le_NumCheque"/>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="Lb_DateCheque">
        <property name="text">
         <string>Date du chèque :</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="Bt_Effacer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Tout effacer</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="Bt_ImprimerDevis">
        <property name="text">
         <string>Imprimer devis</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QComboBox" name="CBx_ModePaiement">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QPushButton" name="Bt_AjoutAnimation">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Ajouter un autre élement (animation, ludimanche...)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QCommandLinkButton" name="Bt_SupprimerItemPanier">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ne plus utiliser la carte de paiement choisie&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ImagesIcones.qrc">
          <normaloff>:/Supprimer.png</normaloff>:/Supprimer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="5" rowspan="8" colspan="2">
       <widget class="QGroupBox" name="GB_Collectivite">
        <property name="title">
         <string>Collectivité</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="5">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QCheckBox" name="CB_PaiementDiffere">
           <property name="text">
            <string>Paiement différé</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Numéro de facture :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="SB_NumFacture"/>
         </item>
         <item row="2" column="4">
          <widget class="QSpinBox" name="SB_AnneeFacture"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
